tests/HappyEyeBallsConnectionBuilderTest.php

```diff
@@ -107,6 +110,8 @@ public function testConnectWillRejectWhenBothDnsLookupsRejectWithDifferentMessag
         $this->assertEquals('Connection to tcp://reactphp.org:80 failed during DNS lookup. Last error for IPv6: DNS6 error. Previous error for IPv4: DNS4 error', $exception->getMessage());
         $this->assertEquals(0, $exception->getCode());
         $this->assertInstanceOf('RuntimeException', $exception->getPrevious());
+
+        Block\await($promise, Loop::get(), 1);
```

Comment:
The exception catching was probably added later so I'll remove line 97. 98, 113, and 114 from this file and others that follow the same pattern. It also looked double when changing the test but wanted it to be in a passing state before looking into some clean up

Reply:
Agree, the test already includes an assertion for a synchronous promise rejection, the `await()` call only be used as a last resort if there's no easy way otherwise :+1:

Reply:
Not sure why I did this, but line 97 and 98 where changed when I initially opened this PR. Should be an easy fix :D

Reply:
Ok dropped most of these, except 4 in another file I'm looking into now. Marking as resolved